packages/schemas/src/player/gamemodes/general/quests/modes/tntgames.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) Statsify
+ *
+ * This source code is licensed under the GNU GPL v3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * https://github.com/Statsify/statsify/blob/main/LICENSE
+ */
+
+import { APIData, removeFormatting } from "@statsify/util";
+import { Field } from "../../../../../metadata";
+import { FormattedGame } from "../../../../../game";
+import { GameQuests } from "../game-quests";
+import { QuestTime, getAmountDuring, questFieldData } from "../util";
+import { add } from "@statsify/math";
+
+export class TNTGamesQuests implements GameQuests {
+  @Field(questFieldData)
+  public winner: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "TNT Run (Daily)" } })
+  public tntRunDaily: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "PVP Run (Daily)" } })
+  public pvpRunDaily: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "Bow Spleef (Daily)" } })
+  public bowSpleefDaily: number;
+
+  @Field({ ...questFieldData, leaderboard: { name: "TNT Tag (Daily)" } })
```

Comment:
The leaderboard metadata won't get spread properly here because you overwrite the leaderboard field.